src/Runtime/Distributions/Automata/ImmutablePairDistribution.cs

```diff
@@ -236,12 +240,6 @@ public override string ToString()
             return base.ToString();
         }
 
-        /// <summary>
-        /// Returns the logarithm of the probability that the current distribution would draw the same sample
-        /// as a given one.
-        /// </summary>
-        /// <param name="that">The given distribution.</param>
-        /// <returns>The logarithm of the probability that distributions would draw the same sample.</returns>
```

Comment:
inheritdoc

Reply:
OK, the fact that without explicit `<inheritdoc/>` the documentation won't be generated based on parent type(s) was news to me. Mostly, because VS was showing doc in tooltips even without `<inheritdoc/>` present.

So, with that in mind I updated doc comments for ImmutablePairDistribution, DiscreteChar, and WeightFunctions.